lib/mongoid_rails_migrations/active_record_ext/migrations.rb

```diff
@@ -338,6 +334,27 @@ def migrate
       end
     end
 
+    def status
+      runnable = runnable_migrations
```

Comment:
Instead of creating this assignment, what about simply using it on line 340? Also, it looks like you could split the to_run variable in to a separate method to keep the status method more focused.